cluster/packages.sls

```diff
@@ -1,6 +1,7 @@
 #required packages to install HA cluster
 
 {%- from "cluster/map.jinja" import cluster with context -%}
+{% set provider = cluster.configure.template.parameters.platform|default('libvirt') %}
```

Comment:
I don't really like this `libvirt` default thing personally. Actually, I don't even like to use `cluster.configure.template.parameters.platform`.
This parameters are used to configure the template (if there is any, it's not mandatory), so I wouldn't like to mix things up.
Can we research a bit if there is any way to know if we are running in azure without getting data from the pillar files?
Sorry for being so strict here, but I want to avoid if we can the usage of parameters in any mode.

Reply:
@diegoakechi Would it be any issue if we install `socat` for all the providers? This would be the easier solution and it would just avoid the need to know the specific provider. I think it's not harmful.

We could add just a comment above saying that is used by `azure` load balancer, and that's it

Reply:
@arbulu89 Not a big problem, apart of customers that are more strict about the package management, and would complain about it. We have on `crmsh` developed a way to identify the cloud providers that could be reused here. Check this function https://github.com/ClusterLabs/crmsh/blob/master/crmsh/utils.py#L2109

Reply:
Ok, maybe as a approach, we can just install `socat` for all the providers by now, and develop this new automatic detection in `crmshmod` for example.
Could you do that @Simranpal ?
Just set a comment above `socat` installation that this is only for azure or something like that

Reply:
Thanks @arbulu89 I remove the platform check for now

Reply:
thanks @arbulu89 I remove the platform check.